auth-api/src/auth_api/services/user.py

```diff
@@ -63,6 +67,65 @@ def as_dict(self):
         obj = user_schema.dump(self._model, many=False)
         return obj
 
+    @staticmethod
+    def create_user_and_add_membership(memberships: List[dict], org_id, token_info: Dict = None,
+                                       skip_auth: bool = False):
+        """
+        Save a user to database and keycloak.
+
+           First add the user to keycloak
+           save the user to DB
+           create membership
+        """
+        if skip_auth:  # make sure no bulk operation and only owner is created using if no auth
+            if len(memberships) > 1 or memberships[0].get('membershipType') not in [OWNER, ADMIN]:
+                raise BusinessException(Error.INVALID_INPUT, None)
+        else:
+            check_auth(org_id=org_id, token_info=token_info, one_of_roles=(ADMIN, OWNER))
+
+        org = OrgModel.find_by_org_id(org_id)
+        if not org or org.access_type != AccessType.ANONYMOUS.value:
+            raise BusinessException(Error.INVALID_INPUT, None)
+
+        current_app.logger.debug('create_user')
+        users = []
+        for membership in memberships:
+            create_user_request = KeycloakUser()
+            username = membership['username']
+            create_user_request.user_name = username
+            create_user_request.password = membership['password']
+            try:
+                # TODO may be this method itself throw the business exception
```

Comment:
Are we going to do this? Can we do it now?

Reply:
Let me see..Not sure if its gonna affect any other functionalities.will come back on this